meson.build

```diff
@@ -29,6 +29,26 @@ if api_bus == 'user'
    common_cflags += '-DUSE_USER_API_BUS'
 endif
 
+# Paths configuration according to used prefix
+path_conf = configuration_data()
+
+prefixdir = get_option('prefix')
+
+sysconfdir = get_option('sysconfdir')
+if not sysconfdir.startswith('/')
+    # sysconfdir is relative to the prefix
+    sysconfdir = prefixdir / get_option('sysconfdir')
```

Comment:
This is a bit overcomplex, this should be enough:
```
sysconfdir = join_paths(prefix, get_option('sysconfdir'))
```

Reply:
As in, join_paths returns the second arg if it is an absolute path already.

Reply:
Fixed